rules/config

```diff
@@ -117,7 +117,7 @@ DEFAULT_VS_PREPARE_MEM = yes
 
 
 # ENABLE_ICCPD - build docker-iccpd for mclag support
-ENABLE_ICCPD = n
+ENABLE_ICCPD = y
```

Comment:
This is required for the docker to build.
I will raise another PR to disable ICCPD once this is merged.

Reply:
you can keep it to n. since the build is passing.

Reply:
Have pushed a commit to restore it back.